CONTRIBUTING.md

```diff
@@ -87,21 +87,14 @@ You can run the entire test suite using `pnpm test`. If you wish to only run e2e
 
 By default, `pnpm test:int` will only run int test against MongoDB. To run int tests against postgres, you can use `pnpm test:int:postgres`. You will have to have postgres installed on your system for this to work.
 
-### Commits
+### Pull Request Titles
 
 We use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for our commit messages. Please follow this format when creating commits. Here are some examples:
```

Comment:
Mention PR squashing here and how the title is used as the commit message, therefore it needs to follow conventional commits.